### PR TITLE
#24 | Connection to MongoDB Server

### LINKED ISSUE #24: Setup and Link Database in MongoLabs
Set up the database in MongoLabs and link it with the Nodejs server.
mlabs Quick guide: https://docs.mlab.com/ 

Ask the [gitter channel](https://gitter.im/JBossOutreach/lead-management) or comment here if you have any doubts.

**4 instances are being created for this task. Only one will be merged but if work is good enough, the task will be accepted even if not merged.**

### PR DESCRIPTION
**Pull request by Khukhuna**
Closes #24

### CODE REVIEW

File: configs/database.config.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+// This fodler should be added in .gitignore
```
Comment (divyanshkul) [inline configs/database.config.js]: Make it **folder**, rest seems fine

Comment (Khukhuna) [reply configs/database.config.js]: Done :smiley:

Comment (Khukhuna) [reply configs/database.config.js]: Oh sorry i changed the text on wrong branch.

File: configs/database.config.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+// This folder should be added in .gitignore
+// But we can leave it for now for testing purposes.
+module.exports = {
+  url: "mongodb://test:test123@ds239873.mlab.com:39873/lead-management-server"
```
Comment (aashutoshrathi) [inline configs/database.config.js]: Make sure you never push Mongo URL directly as it is formed with bare DB credentials, always decouple things and call from .env
Rest LGTM :wink: 
Keep it Up @Khukhuna

Comment (Khukhuna) [reply configs/database.config.js]: Updated pull request with that change. :smile: 
I will take that into account & definitely use in other pull requests too.
Thank you

**Pull request closed without merging**